lighthouse-core/config/config.js

```diff
@@ -540,6 +540,8 @@ class Config {
     }).filter(pass => {
       // remove any passes lacking concrete gatherers, unless they are dependent on the trace
       if (pass.recordTrace) return true;
+      // Always keep defaultPass
+      if (pass.passName === 'defaultPass') return true;
```

Comment:
> I'm still partial to checking if they requested the `devtoolsLog` artifact for keeping it

yeah, that would have gotten us closer to where we need to be wrt to ditching `defaultPass`.

Also needs a test :P